crates/bevy_pbr/src/render/pbr_lighting.wgsl

```diff
@@ -207,9 +207,9 @@ fn point_light(
     // Comes after specular since its NoL is used in the lighting equation.
     L = normalize(light_to_frag);
     H = normalize(L + V);
-    NoL = saturate(dot(N, L));
-    NoH = saturate(dot(N, H));
-    LoH = saturate(dot(L, H));
+    NoL = clamp(saturate(dot(N, L)), 0.0, 1.0);
+    NoH = clamp(saturate(dot(N, H)), 0.0, 1.0);
+    LoH = clamp(saturate(dot(L, H)), 0.0, 1.0);
```

Comment:
[Filament docs](https://google.github.io/filament/Filament.md.html#materialsystem/standardmodelsummary) and the [khronos GLTF sample viewer](https://github.com/KhronosGroup/glTF-Sample-Viewer/blob/47a191931461a6f2e14de48d6da0f0eb6ec2d147/source/Renderer/shaders/pbr.frag#L202) both seem to clamp all of these, so I just added clamps to them to be safe.

Reply:
Removed as they're redundant